docs/cudf/source/user_guide/10min.ipynb

```diff
@@ -2,20 +2,25 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    wait(results_ddf)
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cudf/pull/12114/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='177'/>

Reply:
Do we want to show the futures? I can do so, but maybe I should add some explanation.

Reply:
<p>Not necessary here, My suggestion was to replace <code>wait(results_ddf);</code> with <code>wait(results_ddf)</code>   </p>

Reply:
Ah, the semicolon is the thing that suppresses the output. I just removed it and put the `repr` of the return value in.